go/test/endtoend/vtgr/test_config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "MySQLTopologyUser": "orc_client_user",
+  "MySQLTopologyPassword": "orc_client_user_password",
```

Comment:
We reuse how `vtorc` setup DB connections, and we need a user/password to set up the connection. Since VTGR is also an orchestration component and `orc_client_user` has all the privileges we need - that's why we use `orc_client` here instead of making a new one.